pootle/core/views.py

```diff
@@ -599,18 +599,28 @@ def get(self, *args, **kwargs):
 
     def get_context_data(self, *args, **kwargs):
         filters = {}
+        can_translate = False
         if self.has_vfolders:
             filters['sort'] = 'priority'
 
-        url_action_continue = self.object.get_translate_url(
-            state='incomplete',
-            **filters)
-        url_action_fixcritical = self.object.get_critical_url(
-            **filters)
-        url_action_review = self.object.get_translate_url(
-            state='suggestions',
-            **filters)
-        url_action_view_all = self.object.get_translate_url(state='all')
+        if self.request.user.is_superuser or self.language:
+            can_translate = True
+            url_action_continue = self.object.get_translate_url(
+                state='incomplete',
+                **filters)
+            url_action_fixcritical = self.object.get_critical_url(
+                **filters)
+            url_action_review = self.object.get_translate_url(
+                state='suggestions',
+                **filters)
+            url_action_view_all = self.object.get_translate_url(state='all')
+        else:
+            if self.project:
+                can_translate = True
+            (url_action_continue,
+             url_action_fixcritical,
+             url_action_review,
+             url_action_view_all) = [None] * 4
```

Comment:
This definitely works, however these being just 4 variables, I'd suggest to avoid the fanciness and use regular assignments.